apina-core/src/main/resources/typescript/runtime-common.ts

```diff
@@ -77,6 +77,12 @@ export class ApinaConfig {
             const elementSerializer = this.lookupSerializer(elementType);
             return arraySerializer(elementSerializer);
         }
+
+        const dictionaryMatched = /Dictionary<(.+)>/.exec(type);
```

Comment:
Ei tätä kyllä näin jätetä

Reply:
Ainakin se pitää korjata, että tämä matchaa stringiin `fooDictionary<Bar>Baz`.

Reply:
Muutettu `/^Dictionary<(.+)>$/.`